pre-processing/primod/ribamod/exchange_creator.py

```diff
@@ -0,0 +1,178 @@
+import numpy as np
+import pandas as pd
+import xarray as xr
+from numpy.typing import NDArray
+from scipy.spatial import KDTree
+
+from primod.typing import Bool, Float, Int
+
+
+def _check_conductance(conductance: xr.DataArray) -> xr.DataArray:
+    # Check that the conductance location does not change over time.
+    # The imod_coupler has static weights: changing locations means having
+    # to update the coupling weights over time as well.
+    if "time" in conductance.dims:
+        n_active = conductance.groupby("time").count(xr.ALL_DIMS)
+        if not (n_active == n_active[0]).all():
+            raise ValueError(
+                "For imod_coupler, the number of active cells (defined by the "
+                "conductance) in a river or drainage package must be constant "
+                f"over time. Found:\n{n_active}"
+            )
+        return conductance.isel(time=0, drop=True)
+    else:
+        return conductance
```

Comment:
It doesn't really check doesn't it?
Maybe there is a better fitting name for this function

Reply:
Good point: it only checks that the number is total.

If that fails, another check should be added.

Reply:
In the end, it ensures that there's no time dimension.

How about `_ensure_time_invariant_conductance`?